Source/Tools/LinuxEmulation/LinuxSyscalls/Seccomp/BPFEmitter.cpp

```diff
@@ -0,0 +1,397 @@
+// SPDX-License-Identifier: MIT
+/*
+$info$
+tags: LinuxSyscalls|syscalls-shared
+$end_info$
+*/
+
+#include "LinuxSyscalls/Seccomp/BPFEmitter.h"
+#include "LinuxSyscalls/Seccomp/SeccompEmulator.h"
+
+#include <linux/bpf_common.h>
+#include <linux/filter.h>
+#include <linux/seccomp.h>
+
+namespace FEX::HLE {
+template<bool CalculateSize>
+uint64_t BPFEmitter::HandleLoad(uint32_t BPFIP, const sock_filter* Inst) {
+  [[maybe_unused]] size_t OpSize {};
+  switch (BPF_SIZE(Inst->code)) {
```

Comment:
This switch would be probably better as:

```
if (BPF_SIZE() != W) {
   RET_ERR(EINVAL);
}
```

Reply:
Used the `VALIDATE` define